docs: tidy up Clef Clique signing page (#25527)
ethereum/go-ethereum: docs/_clef/CliqueSigning.md

INFO [06-16|11:14:54.125] Successfully wrote genesis state         database=lightchaindata hash=187412..4deb98
```

> At this point a Clique network exists with blocks that can be signed.

Reviewer: ```suggestion
At this point a Clique network exists with blocks that can be signed.
```
Well, we've initiated geth with a geneiss, but no blocks exist yet.

Author: yes ofc - thanks!

```suggestion
At this point a Geth has been initiated with a genesis configuration.
```

## Prepping Clef

> In order to make use of Clef for signing:

Reviewer: ```suggestion
In order to make use of `clef` for signing:
```
When we talk about the binary itself, I think we should use `clef`. I may have used "Clef" here and there, but that's not right IMO

Author: ok!

> 1. Ensure Clef knows the password for the keystore.

Author: ```suggestion
1. Ensure `clef` knows the password for the keystore.
```

> 2. Ensure Clef auto-approves clique signing requests.

Author: ```suggestion
2. Ensure `clef` auto-approves clique signing requests.
```

> These two things are independent of each other. First of all, however, Clef must be initiated (for this example the password is `clefclefclef`)

Author: ```suggestion
These two things are independent of each other. First of all, however, `clef` must be initiated (for this example the password is `clefclefclef`)
```

Decrypt master seed of clef
Password: 
INFO [06-16|11:27:09.153] Credential store updated                 set=0x9CD932F670F7eDe5dE86F756A6D02548e5899f47
```

At this point, if Clef is used as a sealer, each block would require manual approval, but without needing to provide the password.

### Testing stored password

> To test that the stored password is correct and being properly handled by Clef, first start Clef:

Author: ```suggestion
To test that the stored password is correct and being properly handled by Clef, first start `clef`:
```

    console.log(JSON.stringify(r))
}
```
In order to use a certain ruleset, it must first be 'attested'. This is to prevent someone from modifying a ruleset-file on disk after creation.

```sh
$ clef --keystore ./ddir/keystore --configdir ./clef --chainid 15 --suppress-bootwarn  attest  `sha256sum rules.js | cut -f1`
```
which returns:

```terminal
Decrypt master seed of clef
Password: 
INFO [06-16|13:49:00.298] Ruleset attestation updated              sha256=54aae496c3f0eda063a62c73ee284ca9fae3f43b401da847ef30ea30e85e35d1
```

> And Clef can be started, pointing out the `rules.js` file. 

Author: ```suggestion
And `clef` can be started, pointing out the `rules.js` file. 
```

```sh
$ clef --keystore ./ddir/keystore --configdir ./clef --chainid 15  --suppress-bootwarn  --rules ./rules.js 
```

> Once Geth starts asking Clef to seal blocks, the data will be displayed. From that data, rules can be defined that allow signing clique headers but nothing else.

Author: ```suggestion
Once Geth starts asking `clef` to seal blocks, the data will be displayed. From that data, rules can be defined that allow signing clique headers but nothing else.
```

  0000000000000000,
]
```

> However, `messages` could also be used. They do not come from the external caller, but are generated from the Clef internals: Clef parsed the incoming request and verified the Clique wellformedness of the content. The following simply checks for such a message:

Author: ```suggestion
However, `messages` could also be used. They do not come from the external caller, but are generated inernally: `clef` parsed the incoming request and verified the Clique wellformedness of the content. The following simply checks for such a message:
```

	return "Reject"
}
```

Attest the ruleset:

```sh
$ clef --keystore ./ddir/keystore --configdir ./clef --chainid 15 --suppress-bootwarn  attest  `sha256sum rules.js | cut -f1`
```

returning

```terminal
Decrypt master seed of clef
Password: 
INFO [06-16|14:18:53.476] Ruleset attestation updated              sha256=7d5036d22d1cc66599e7050fb1877f4e48b89453678c38eea06e3525996c2379
```

> Run clef:

Author: ```suggestion
Run `clef`:
```

```sh
$ clef --keystore ./ddir/keystore --configdir ./clef --chainid 15  --suppress-bootwarn  --rules ./rules.js 
```
Run Geth:

```sh
$ geth  --datadir ./ddir --signer ./clef/clef.ipc --mine
```

> And Clef should now happily sign blocks:

Author: ```suggestion
And `clef` should now happily sign blocks:
```

  --suppress-bootwarn --signersecret ./clefpw \
    attest  `sha256sum rules.js | cut -f1`
```

> The normal startup command for Clef:

Author: ```suggestion
The normal startup command for `clef`:
```
